packages/next-swc/crates/next-core/src/next_import_map.rs

```diff
@@ -810,11 +813,20 @@ async fn insert_next_shared_aliases(
         ImportMapping::Dynamic(Vc::upcast(NextFontGoogleReplacer::new(project_path))).into(),
     );
 
+    let proxy_option = match std::env::var("http_proxy") {
+        Ok(proxy) => Vc::cell(Some(ProxyConfig::Http(proxy))),
+        Err(_) => match std::env::var("https_proxy") {
+            Ok(proxy) => Vc::cell(Some(ProxyConfig::Https(proxy))),
+            Err(_) => Vc::cell(None),
+        },
+    };
```

Comment:
This would break persistent caching since the env var might change between runs.
Use turbo-tasks-env to read env vars.